viper/common/objects.py

```diff
@@ -105,11 +105,7 @@ def __init__(self, path):
         self.children = ''
 
         if self.is_valid():
-            if sys.version_info < (3, 0):
-                # PY2SUPPORT
-                self.name = os.path.basename(self.path).encode('utf-8')
-            else:
-                self.name = os.path.basename(self.path)
+            self.name = os.path.basename(self.path)
```

Comment:
@Rafiot could you please have a look at this. I ran unittest on Py35 and Py27 and the Py2 tests fail on the ".encode('utf-8')" command. Removing it fixes this and the tests work for both 2+3. So it seems this differentiation is not needed - or there is something wrong with the Unitttests... :-)